src/GraphQL/Types/Scalars/EnumerationGraphType.cs

```diff
@@ -140,16 +140,41 @@ public EnumerationGraphType()
                 name: ChangeEnumCase(e.name),
                 value: Enum.Parse(type, e.name),
                 description: e.member.Description(),
-                deprecation: e.member.ObsoleteMessage()
+                deprecation: e.member.ObsoleteMessage(),
+                member: e.member
             ));
 
             Name = type.Name.ToPascalCase();
             Description ??= typeof(TEnum).Description();
             DeprecationReason ??= typeof(TEnum).ObsoleteMessage();
 
-            foreach (var (name, value, description, deprecation) in enumGraphData)
+            foreach (var (name, value, description, deprecation, member) in enumGraphData)
             {
-                AddValue(name, description, value, deprecation);
+                var enumValue = new EnumValueDefinition
+                {
+                    Name = name,
+                    Value = value,
+                    Description = description,
+                    DeprecationReason = deprecation,
+                };
+                bool ignore = false;
+                foreach (var attr in member.GetCustomAttributes<GraphQLAttribute>())
+                {
+                    if (!attr.ShouldInclude(member, true))
```

Comment:
true/false? null? for second argument

Reply:
As a scalar is both input and output equally. Do you think I should make the property a nullable bool or just write in the xml comment that it’s true for enums?

Reply:
Changing to null.

Reply:
ok